src/Main.java

```diff
@@ -1,37 +1,89 @@
 import java.util.Scanner;
 
 public class Main {
-
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+
+        System.out.println("The hero has entered an enchanted forest full of adventures");
+
+
+        int health = 100;
+        int luck = 50;
+
+
+        System.out.println("\nEvent 1: Crossroads");
+        System.out.println("\nChoose a road:");
+        System.out.println("1 - old trail (-20 health)");
```

Comment:
Not a mistake, but I'd like to play the game without spoilers -> now we're giving away what will happen when I choose something

I want to make choices without knowing the potential result, as it makes this game more interesting, right? :)